api/src/database/dbClient.ts

```diff
@@ -0,0 +1,6 @@
+import Knex from 'knex';
+import { config } from '../config';
+
+let _instance: Knex;
+
+export const getDbClientInstance = () => _instance || (_instance = Knex(config.database));
```

Comment:
very nice! I'd put a comment above this explaining what you're doing (lazy initialization) as that's not super-duper readable IMHO